WeakAuras/Prototypes.lua

```diff
@@ -6215,6 +6215,47 @@ WeakAuras.event_prototypes = {
     automaticrequired = true
   },
 
+  ["Current Action"] = {
```

Comment:
I think we need a better name.
Discord search is broken, but we should look up what terminology people are using when they ask for that functionality.

Reply:
It seems "queued" is the right word?

Reply:
Actually probably best to change this here too, as long as we can still do that.

(Note, also need to change the    WeakAuras.event_prototypes["Current Action"] = nil line)